src/constants/widgetSettings.ts

```diff
@@ -1,7 +1,6 @@
 // Copyright 2020 Cognite AS
```

Comment:
will the tool widget be shown in the widget configurator where we allow users to find widget by sizes?

Reply:
No